pyspecdata/DCCT_function.py

```diff
@@ -261,37 +274,45 @@ def DCCT(
         A = this_nddata.C
         A.rename(a_shape.dimlabels[:-2][0], "smooshed")
 
+    # }}}
+    # {{{ vertical lines for coh paths
     def draw_span(
         ax1,
         ax2,
         label,
         this_label_num,
-        allow_for_text=allow_for_text_default,
-        allow_for_ticks=allow_for_ticks_default,
     ):
-        label_spacing = this_label_num * label_spacing_multiplier
-        # {{ transform from axes coords to figure coords for respective axes
-        ax1_to_figure = ax1.transAxes + fig.transFigure.inverted()
-        ax2_to_figure = ax2.transAxes + fig.transFigure.inverted()
+        """Place the vertical lines and labels of the coherence transfer
+        pathways"""
+        label_spacing = (
+            this_label_num
+            + 1  # plus one so the first horizontal isn't placed at 0
+            #      (overlapping with the spine of the indirect axis)
+        ) * vert_label_spacer  # will space the vertical lines along x approp.
+        x1_disp = LHS_labels + bbox[0] - label_spacing  # x coord is the left
+        #                                                 side of the axis
+        #                                                 minus the spacing for
+        #                                                 text/ticks
+        #                                                 (label_spacing)
+        # {{{ Take y coordinate of top and bottom of axes objects to get the 2
+        # points for drawing the lines. To be exact I pull this from the axes
+        # objects themselves.
+        _, y1_fig = ax_to_fig(ax1, fig).transform(r_[0, 0.95])
+        _, y2_fig = ax_to_fig(ax2, fig).transform(r_[0, 0.05])
```

Comment:
this is where I use the function that I defined above going from axes coords (easy, (0,0) or(0,1)) to figure coords for the final scaled translation